examples/kitchensink/server.go

```diff
@@ -106,7 +106,7 @@ func NewKitchenSink(channelSecret, channelToken, appBaseURL string) (*KitchenSin
 func (app *KitchenSink) Callback(w http.ResponseWriter, r *http.Request) {
 	cb, err := webhook.ParseRequest(app.channelSecret, r)
 	if err != nil {
-		if err == linebot.ErrInvalidSignature {
+		if err.Error() == linebot.ErrInvalidSignature.Error() {
```

Comment:
Could you try 

//   if err == ErrNotFound { … }
if errors.Is(err, linebot.ErrInvalidSignature) {
    // something wasn't found
}

Reply:
Thank you for your suggestion！

 I've tried using errors.Is() for error comparison, and it works as expected. 
 
 However, it's crucial to use **ErrInvalidSignature from the webhook package**; otherwise, the comparison fails. 

 I've also made corresponding adjustments wherever necessary.